backends-common/elasticsearch-v6/src/test/java/org/apache/james/backends/es/v6/search/ScrollIterableTest.java

```diff
@@ -50,7 +50,7 @@
 
 public class ScrollIterableTest {
 
-    private static final TimeValue TIMEOUT = new TimeValue(6000);
+    private static final TimeValue TIMEOUT = TimeValue.timeValueMinutes(1);
```

Comment:
I think it should be 6 seconds due to the old code `new TimeValue(6000)`

Reply:
6 seconds is rather short - I already noticed that infinite scroll was blocking at some point in OpenPaaS Inbox. Corelated?

I'd rather keep the same constants than in other places.

Reply:
Let's do that in another PR - but I belive `6 seconds` is a bit too short.

Reply:
Let's do that in another PR - but I belive `6 seconds` is a bit too short.

Reply:
Looking at ES 2.x code, the timeoute is `new TimeValue(60000)` hence I'll keep the 1 minute timeout here too.